main/externalData/balances/worker.ts

```diff
@@ -74,9 +83,11 @@ async function tokenBalanceScan (address: Address, tokensToOmit: Token[] = [], c
 
 async function fetchTokenBalances (address: Address, tokens: Token[]) {
   try {
-    const tokenBalances = await balances.getTokenBalances(address, tokens)
+    const blacklistSet = new Set(tokenLoader.getBlacklist([]).map(({chainId, address}) => `${chainId}:${address.toLowerCase()}`))
```

Comment:
doesn't `getTokens()` already filter this? or do we need to check again? either way seems like we're doing this logic a few times across the code, might want to put it in one place and re-use it

Reply:
We don't use the getTokens list in here, it uses the known tokens list

Reply:
Going to create a toTokenId function which converts the Token into a string 👍